statemachine/statemachine.py

```diff
@@ -21,6 +21,10 @@
     from .state import State
 
 
+EMPTY_RETURN_VALUES = (None, [])
```

Comment:
**suggestion:** Consider using a set instead of a tuple.

```suggestion
EMPTY_RETURN_VALUES = {None, []}
```

Reply:
You cannot add a non hashable type (`[]`) to a set.